### DISCUSSION TITLE
using other folder path in app router

### DISCUSSION THREAD

**Discussion by bysxx** [vercel/next.js#64955] [App Router]
I want to use API Routes in App Router, but I want to use a other other folder path than the app folder path, is there any way?

I want to divide the backend source coude and the frontend code :(

**Comment by icyJoseph** [comment]
Wouldn't it be ok to create `app/api/` and put your Route Handlers there?

**Reply by bysxx** [reply]
I want to put it outside the `/app`, is there any way?

**Reply by icyJoseph** [reply]
As far as I know, no. 

Maybe you could do all the function definitions elsewhere and then just re-export in the route handlers... 

Or maybe use route groups, never thought about using those for route handlers, it'd still be in app, but clearly separated from the rest. 

Or use Pages router for these instead, but that's a cop-out, don't do that

**Reply by bysxx** [reply]
Thank you!
If I find how to do it, I'll comment here.
